config/default/kustomization.yaml

```diff
@@ -1,5 +1,5 @@
 # Adds namespace to all resources.
-namespace: grafana-operator-experimental-system
```

Comment:
Since we will merge in to the grafana-operator it will just create issues if we call it experimental when we are developing. Especially if we will write e2e tests etc.